code/game/objects/structures/flora.dm

```diff
@@ -365,4 +365,21 @@
 /obj/structure/flora/pottedplant/deskferntrim
 	name = "fancy trimmed ferny potted plant"
 	desc = "This leafy desk fern seems to have been trimmed too much."
-	icon_state = "plant-30"
\ No newline at end of file
+	icon_state = "plant-30"
+
+// Space/Asteroid Flora
+
+/obj/structure/flora/shimmering_orb
+	name = "shimmering orb"
+	desc = "A floating stone shaped in the form of an orb, small cracks of bioluminescent growths within give off a calming light."
+	icon = 'icons/urist/asteroidflora.dmi'
+	icon_state = "shimmering_orb"
+	plane = ABOVE_HUMAN_PLANE
+	layer = ABOVE_HUMAN_LAYER
+
+/obj/structure/flora/shimmering_orb/Initialize()
+	. = ..()
+	set_light(1, 1, 4, 2, "#0066ff")
+	light_inner_range = 5
+	light_max_bright = 1
+	light_outer_range = 3
```

Comment:
set_light()'s entire purpose is to do this stuff. It doesn't make much sense to set these values twice.
Side note, inner range being greater than outer range is equally strange.